src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +202,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        processUserInput();
+    }
+
+	private static void processUserInput() {
+		while (true) {
```

Comment:
Good job on extracting a method out to process user input. Remember that in the commit message, mention which refactoring you applied. For example, Extract method processUserInput()